lib/serve/rest-api/src/api/endpoints/v2/litellm_passthrough.py

```diff
@@ -0,0 +1,64 @@
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Model invocation routes."""
+
+import logging
+from collections.abc import Iterator
+from typing import Union
+
+import requests
+from fastapi import APIRouter, Request
+from fastapi.responses import JSONResponse, Response, StreamingResponse
+
+# Local LiteLLM installation URL
+LITELLM_URL = "http://localhost:4000"
```

Comment:
Does this need to be more dynamic? If not should we add a note in the entrypoint script that if the port is changed for litellm this also needs to be updated?

Reply:
I can write that comment- this is the only thing running alongside the FastAPI, and since the port isn't exposed, I'm not expecting anything to call it directly or have a need for it to be changed.